From 54e26134e3010889aec435a6bcb2c3525d9d1643 Mon Sep 17 00:00:00 2001
From: Carsten Igel <1760987+carstencodes@users.noreply.github.com>
Date: Sat, 10 Feb 2024 16:53:42 +0100
Subject: [PATCH] deploy: Fix installation on docker

Failed to build  docker image for pixelfed
v0.11.11 as there are issues with installation
of libwebp7 and composer

Updating docker base image to PHP 8.2 and debian
bookworm helps this
---
 contrib/docker/Dockerfile.apache | 2 +-
 contrib/docker/Dockerfile.fpm    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/docker/Dockerfile.apache b/contrib/docker/Dockerfile.apache
index a400f8797b..0fc27fbfeb 100644
--- a/contrib/docker/Dockerfile.apache
+++ b/contrib/docker/Dockerfile.apache
@@ -1,4 +1,4 @@
-FROM php:8.1-apache-bullseye
+FROM php:8.2-apache-bookworm
 
 ENV COMPOSER_MEMORY_LIMIT=-1
 ARG DEBIAN_FRONTEND=noninteractive
diff --git a/contrib/docker/Dockerfile.fpm b/contrib/docker/Dockerfile.fpm
index 1bb0a15f7a..0a548c6a7e 100644
--- a/contrib/docker/Dockerfile.fpm
+++ b/contrib/docker/Dockerfile.fpm
@@ -1,4 +1,4 @@
-FROM php:8.1-fpm-bullseye
+FROM php:8.2-fpm-bookworm
 
 ENV COMPOSER_MEMORY_LIMIT=-1
 ARG DEBIAN_FRONTEND=noninteractive
